lib/moonshot/stack.rb

```diff
@@ -291,17 +291,22 @@ def wait_for_stack_state(wait_target, past_tense_verb)
       result
     end
 
-    def make_tags
+    def self.make_tags(config)
       default_tags = [
-        { key: 'moonshot_application', value: @config.app_name },
-        { key: 'moonshot_environment', value: @config.environment_name }
+        { key: 'moonshot_application', value: config.app_name },
+        { key: 'moonshot_environment', value: config.environment_name },
       ]
+      name=[config.app_name, config.environment_name].join('-')
```

Comment:
line 18 and this are duplicated. To maintain DRY, I think this should be wrapped in its own method.
If someone changes line 18 and misses this line that creates inconsistencies.

Reply:
had to make it a class method instead of an instance, but did it